incubator/istio/values.yaml

```diff
@@ -62,18 +67,6 @@ pilot:
           cpu: 100m
           memory: 128Mi
```

Comment:
The recommendation is not to specify defaults for resources.

Reply:
Okay. Will remove